client/tests/webdriver/pages/report/createReport.page.js

```diff
@@ -121,10 +121,6 @@ class CreateReport extends cr.CreateReport {
     await (
       await this.getReportPeopleTable()
     ).waitForExist({ reverse: true, timeout: 3000 })
-    if (!interlocutor) {
-      const { interlocutorCheckbox } = await this.getPersonByName(name)
-      await interlocutorCheckbox.click()
-    }
```

Comment:
By removing this you are now no longer testing moving an attendee from the interlocutor to the advisor section. Please reinstate a test for this.